src/frontend/src/pages/CreateChangeRequestPage/CreateChangeRequestView.tsx

```diff
@@ -105,6 +105,21 @@ const CreateChangeRequestsView: React.FC<CreateChangeRequestViewProps> = ({
   const projectOptions: { label: string; id: string }[] = [];
   const wbsDropdownOptions: { label: string; id: string }[] = [];
 
+  const customComparator = (option1: { id: string }, option2: { id: string }) => {
```

Comment:
this method is a bit overfitted to the use-case, I think we could actually end up using this in a lot of different places. Rename it to wbsNumComparator and have it in two strings, wbsNum1, wbsNum2; Also move it to src\shared\src\validate-wbs.ts